app/assets/images/pulsing_location_marker.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
-  <circle cx="8" cy="8" r="8" fill="#09a8f0"/>
-</svg>
\ No newline at end of file
+<svg width="50" height="50" xmlns="http://www.w3.org/2000/svg">
+  <circle transform="translate(19,19)" cx="6" cy="6" r="6" fill="#09a8f0"/>
```

Comment:
translate moves the circle to the middle of the svg
formula (width / 2) - radius = (50 / 2) - 6) = 19